server/pub/permissions.ts

```diff
@@ -23,6 +23,7 @@ const managerUpdatableFields = [
 	'nodeLabels',
 	'pubEdgeListingDefaultsToCarousel',
 	'pubEdgeDescriptionVisible',
+	'reviewHash',
```

Comment:
let me think for a sec...i was trying to set a reviewHash on the client if it doesnt exist. but then i changed that to be on the server, so a review hash is ensured for a pub. so yeah this can go bc these are fields that be can be updated by users